src/Action/PostCollectionAction.php

```diff
@@ -42,8 +42,9 @@ public function __construct(SerializerInterface $serializer)
      */
     public function __invoke(Request $request)
     {
-        list($resourceClass, $operationName, , $format) = $this->extractAttributes($request);
-        $context = ['resource_class' => $resourceClass, 'collection_operation_name' => $operationName];
+
+        list($resourceClass, $operationName,$itemOperation, $format) = $this->extractAttributes($request);
```

Comment:
> It does trigger an syntax error with php7.0.4

How is that possible? Travis tests are using latest PHP 7, which is 7.0.4, and the tests are passing.